protocol/communities/manager.go

```diff
@@ -513,6 +518,61 @@ func (m *Manager) runENSVerificationLoop() {
 	}()
 }
 
+// This function is mostly a way to fix any community that is missing tokens in its description
+func (m *Manager) fillMissingCommunityTokens() error {
```

Comment:
To me, this function looks hard to predict knowing that it's called in different go routines, but seems ok to me as well. I thought about using go channels as well, but it would require some deeper analysis.